src/components/Modals/OnboardingTransfer.vue

```diff
@@ -0,0 +1,562 @@
+      displayOptions.shown = !contentShown;
+      if (contentShown) {
+        displayOptions.boxBorder = '#C4C4C4';
+        displayOptions.arrowColor = '#C4C4C4';
+      } else {
+        displayOptions.boxBorder = '#32A0F2';
+        displayOptions.arrowColor = '#32A0F2';
+      }
+    },
+    showHideExamContent(i) {
+      this.showHideContent('exam', 'type', i);
+    },
+    showHideSubjectContent(i) {
+      this.showHideContent('exam', 'subject', i);
+    },
+    showHideScoreContent(i) {
+      this.showHideContent('exam', 'score', i);
+    },
+    showHideClassContent(i) {
+      this.showHideContent('class', '', i);
+    },
+    closeDropdownIfOpen(section, i) {
+      let displayOptions = this.displayOptions[section];
+      displayOptions = displayOptions[i];
+      if (section === 'exam') {
+        Object.keys(displayOptions).forEach(key => {
+          if (key !== 'equivCourse' && displayOptions[key].stopClose) {
+            displayOptions[key].stopClose = false;
+          } else if (key !== 'equivCourse' && displayOptions[key].shown) {
+            displayOptions[key].shown = false;
+            displayOptions[key].boxBorder = '#C4C4C4';
+            displayOptions[key].arrowColor = '#C4C4C4';
+          }
+        });
+      } else if (displayOptions.stopClose) {
+        displayOptions.stopClose = false;
+      } else if ('equivCourse' && displayOptions.shown) {
+        displayOptions.shown = false;
+        displayOptions.boxBorder = '#C4C4C4';
+        displayOptions.arrowColor = '#C4C4C4';
+      }
+    },
+    closeTypeDropdownIfOpen(event, i) {
+      this.closeDropdownIfOpen('exam', i);
+    },
+    closeClassDropdownIfOpen(event, i) {
+      this.closeDropdownIfOpen('class', i);
+    },
+    // Set the colleges map to with acronym keys and full name values
+    setExamsMap() {
+      /** @type {Object.<string, string>} */
+      const exams = [];
+      Object.keys(reqsData).forEach(key => {
+        exams.push(key);
+      });
+      this.exams = exams;
+    },
+    // Set the majors map to with acronym keys and full name values
+    setSubjectList() {
+      /** @type {Object.<string, string>} */
+      const totalSubjects = [];
+      this.displayOptions.exam.forEach(exam => {
+        if (exam.type.placeholder !== placeholderText) {
+          const examType = exam.type.placeholder;
+          const subjects = [];
+          if (examType in reqsData && examType !== null) {
+            reqsData[examType].forEach(sub => {
+              subjects.push(sub.subject);
+            });
+            totalSubjects.push(subjects);
+            if (examType === 'AP') {
+              this.scores = [1, 2, 3, 4, 5];
+            } else {
+              this.scores = [1, 2, 3, 4, 5, 6, 7];
+            }
+          }
+        }
+      });
+      this.subjects = totalSubjects;
+    },
+    // Didn't want to seperate into two functions but v-model wouldn't work unless clicked twice?
+    updateSwimYes() {
```

Comment:
Want to elaborate on this? Might be able to help - but also I think I duplicated a lot of the original onboarding code because I was having trouble with stuff like this

Reply:
I'm going to leave it as is since it sounds like this issue happened to you too?